sdk/java-sdk/src/main/scala/com/akkaserverless/javasdk/impl/AnySupport.scala

```diff
@@ -389,8 +389,13 @@ class AnySupport(
         ScalaPbAny(BytesPrimitive.fullName, primitiveToBytes(BytesPrimitive, byteString))
 
       case other =>
-        throw SerializationException(
-          s"Don't know how to serialize object of type ${other.getClass}. Try passing a protobuf, using a primitive type, or using a type annotated with @Jsonable.")
+        try {
+          ScalaPbAny.fromJavaProto(JsonSupport.encodeJson(value))
+        } catch {
+          case _: IllegalArgumentException =>
+            throw SerializationException(
+              s"Don't know how to serialize object of type ${value.getClass}. Try passing a protobuf, using a primitive type, or using a type annotated with @Jsonable.")
+        }
```

Comment:
I'm not seeing deserialization anywhere in this PR, how is json from the proxy turned into the pojo instances when starting the entity (and replaying events)?

Reply:
I don't know! 🤷 
This still need to be verified. 

Seems to work or the code path is not being exercised. Most probably the latter. 
The integration test start from scratch and the entity stays in memory during the test so there is not replay or load of any state. That's probably why it works. 

I'm aware of this and will investigate this further.